content/fr/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Concepts sous-jacents à Cloud Controller Manager
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Le concept de cloud controller manager (CCM, à ne pas confondre avec le binaire) a été créé à l'origine pour permettre au code de providers cloud et au noyau Kubernetes d'évoluer indépendamment les uns des autres. Le cloud controller manager fonctionne aux côtés d'autres composants principaux, tels que le Kubernetes controller manager, l'API server et le scheduler. Il peut également être démarré en tant qu’addon Kubernetes, auquel cas il s’applique au-dessus de Kubernetes.
+
+La conception du cloud controller manager repose sur un mécanisme de plugin qui permet aux nouveaux providers cloud de s'intégrer facilement à Kubernetes à l'aide de plugins. Des mesures sont en place pour l'intégration de nouveaux providers cloud sur Kubernetes et pour la migration de l'ancien modèle vers le nouveau modèle CCM.
```

Comment:
suggest: fournisseurs de clouds

Reply:
Do you say that in your work ? I have always use: "cloud", "cloud provider" or "provider cloud"

Reply:
Do you say that in your work ? I have always use: "cloud", "cloud provider" or "provider cloud"

Reply:
at work i use: "CSP" :), but in a french translated website, "fournisseur de cloud" looks good no? what do you think folks?